assets/js/components/Battery/BatterySettingsModal.vue

```diff
@@ -261,18 +261,20 @@ import formatter, { POWER_UNIT } from "../../mixins/formatter.js";
 import collector from "../../mixins/collector.js";
 import api from "../../api.js";
 import smartCostAvailable from "../../utils/smartCostAvailable.js";
+import { defineComponent, type PropType } from "vue";
+import type { Battery, SelectOption } from "assets/js/types/evcc.js";
 
-export default {
+export default defineComponent({
 	name: "BatterySettingsModal",
 	components: { SmartCostLimit, CustomSelect, GenericModal },
 	mixins: [formatter, collector],
 	props: {
 		bufferSoc: Number,
```

Comment:
```suggestion
		bufferSoc: { type: Number, default: 100 },
```

Wenn der nicht gesetzt ist nehmen wir im UI `100` an. Diese Fallbacks (glaube es sind 2) könnten wir dann auch rauswerfen. Dann ists konsistent und korrekt getypt.